test/es6/HTMLComments.js

```diff
@@ -71,4 +74,4 @@ assert.throws(function () { eval('/* */--> WScript.Echo("Code after /* */--> is
 var a = 1; a-->a; WScript.Echo("Code after post-decrement with a greater-than comparison (-->) is reachable");
 assert.areEqual(0, a, "Post decrement executes");
 
-assert.throws(function () { eval('/* */ --->'); }, SyntaxError, "HTMLCloseComment causes syntax error with an extra -", "Syntax error");
+assert.throws(function () { eval('/* */ --->'); }, SyntaxError, "HTMLCloseComment causes syntax error with an extra -", "Unexpected token '>' after '-'");

```

Comment:
These error messages are perhaps more descriptive of the parser state but probably just as confusing to the author. IMO this is fine though.

Reply:
Yeah I didn't like that one much but wasn't sure what to do with it; the parser has -- i.e. predecrement operator then a - i.e. unary minus then is expecting a statement but hits > I could special case it but would be a bit messy and wasn't sure it was worth it. At least with this option the author knows that it's the "-" and ">" that there's a problem with rather than looking earlier/later in the line.

Reply:
Probably not worth it. I think this is fine.